assets/panel/debugger.properties

```diff
@@ -405,6 +405,12 @@ breakpointMenuItem.removeCondition.label=Remove breakpoint condition
 breakpointMenuItem.removeCondition.accesskey=c
 breakpointMenuItem.editCondition.label=Edit breakpoint condition
 breakpointMenuItem.editCondition.accesskey=n
+breakpointMenuItem.disableAllAtLine.label=Disable breakpoints on line
+breakpointMenuItem.disableAllAtLine.accesskey=Z
```

Comment:
This is going to show up as `Disable breakpoints on line (Z)`. As long as you're OK with that, I don't have an issue.

Note that in preferences we gave up on having unique accesskeys, given the number of controls in in the same scope).